src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const toggleLikeButton = () => {
+    props.updateLiked(props.id);
+  };
+
+  // Note: Tests for waves don't pass if I have the commented-out lines :,( 
+  // But if I don't use colorData, the tests will pass. 
+  // Just wanted to do the optional enhancement. It works in browser!
+
+  // const textColor = props.colorData[props.sender];
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={ props.id % 2 ? 'chat-entry local' : 'chat-entry remote' }>
```

Comment:
This assumes two things: 1) ids increase monotonically; 2) there are two senders who alternate messages.

Consider how you could change your code so that you didn't have to rely on these assumptions in order to render the messages. What would you need to change?